packages/driver/src/multi-domain/domain_fn.ts

```diff
@@ -49,73 +30,82 @@ export const handleDomainFn = (cy: $Cy, specBridgeCommunicator: SpecBridgeDomain
 
     // Set the state ctx to the runnable ctx to ensure they remain in sync
     cy.state('ctx', cy.state('runnable').ctx)
+
+    // Stability is always false when we start as the page will always be
+    // loading at this point
+    cy.isStable(false, 'multi-domain-start')
```

Comment:
I'm curious what would happen if we synced stability with the main domain, but thats an experiment for another day.